### DISCUSSION TITLE
データベースの選定

### DISCUSSION THREAD

**Discussion by A2GMer** [A2GMer/quiz-template#4] [Show and tell]
@A2GMer @tnorip 
# 概要
クライアント側とサーバ側で個別のDBを構築する必要があると思うけど、何のDBにしたらいいだろう・・・
一緒に決めたい！

# 目的
このdiscussionで相談しながら決められたらOK！
将来的に「このDB選定は間違ってたな！」ってなっても、
「こういう考えで決めました！」という考え方をここに残しておくことが目的！

# タスク
- [x] クライアント側のDB決定
- [x] サーバ側のDB決定

**Comment by A2GMer** [comment]
@tnorip 
flutterに[SQLiteのプラグイン(sqflite)](https://pub.dev/packages/sqflite)が内包されているから、

1. まずはクライアント側もサーバ側もsqfliteで作る。
2. その後に、サーバ側は切り出す。

というのが駆け出しとしては早いかなと思うけど、どうでしょう？？

**Reply by tnorip** [reply]
@A2GMer 
調べてみたけど良さそうね！とりあえず形にしたいし
sqfliteで進めていこうか！
